helm-charts/inbucket/values.yaml

```diff
@@ -56,6 +56,7 @@ extraEnv:
 
 ingress:
   enabled: false
+  #className: your-custom-ingress-class
```

Comment:
you can uncomment the field and make that an empty string

```
ingress:
  enabled: false
  className: ""
```

also please sign the DCO

and thanks for this PR

Reply:
Fixed. Thank you.